### DISCUSSION TITLE
Should Finder quit when closing a Bunch?

### DISCUSSION THREAD

**Discussion by isaacniebeling** [ttscoff/bunch-docs#81] [Q&A] [answered]
One of my Bunches includes the following:

```
Finder
- XX
- ~/Documents/<folder>
```

It opens up a Finder window in `<folder>`, but when the Bunch closes, Finder quits as well. To a certain extent, this makes sense based on the "usual" syntax for launching apps and opening windows, but... Finder is kind of weird and I generally expect it to not quit unless something went horribly awry. Should I be doing something different to keep Finder open (I don't mind if those _windows_ close again, just not _all of Finder_.)

This is on 10.14.6 with Beta 1.4.0 (93) (I did _just_ see that there's a new beta out, so I'm installing that now, in case that makes a difference)

**Comment by ttscoff** [comment, marked as answer]
No, Finder probably shouldn't quit, that may be a new bug. That said, you can always use % to prevent any app from quitting on close:

%Finder

- Brett

**Reply by isaacniebeling** [reply]
Yeah, I've added `%` now. I have a vague recollection that it didn't _always_ used to be this way, but I also don't have any idea when it changed (I'm not even 100% sure it _did_ change).

**Reply by ttscoff** [reply]
I've added a universal ignore for Finder when closing for the next update. You can still force Finder to quit on close using `!!!Finder`, but by default it will ignore it when quitting. Also noticed that if Finder isn't running and you try to open a folder, it doesn't launch when opening the Bunch, so I'll try to fix that for the next update as well.

**Reply by isaacniebeling** [reply]
Awesome! Thanks!
